### DISCUSSION TITLE
[FEATURE REQ] add API endpoints to add indexers/feeds

### DISCUSSION THREAD

**Discussion by appiekap653** [autobrr/autobrr#1687] [Ideas]
**Is your feature request related to a problem? Please describe.**
It would be nice to be able to add support for [Prowlarr](https://github.com/Prowlarr/Prowlarr) to automatically sync indexers/feeds into Autobrr.
But to open a feature request for Prowlarr, the API of Autobrr need to have those endpoints first. 

The intent for Prowlarr is to be the one central place to manage (add/edit/delete) indexers. 
It would be nice to leave the responsibility there instead of having to add/edit/delete in Prowlarr first, then again in Autobrr.

This would also give solution to issues #1344 and #961 because the name of the indexers/feeds will get synced properly. 

**Describe the solution you'd like**
Expand the API indexers/ feeds endpoint to support adding from Prowlarr.

**Comment by zze0s** [comment]
Hi!

The endpoints already exists, the webui is all using the API, it's just not very well documented because there has been no need for it. 🙂

Prowlarr indexers and autobrr indexers (torznab ones at least) do have some differences and won't be a 1-1 mapping.

I'm pretty confident that if you open a feature req in Prowlarr they will tell you that **it's on us to implement _their_ API's** to allow for sync, not the other way around.

It would only be beneficial for the indexers that does not have IRC and only rely on Torznab because we use different data/creds etc in many cases and the IRC part would have to be manual either way.

I'm not opposed to this but it's not something I'm planning to work on in the near future.
